go.mod

```diff
@@ -3,34 +3,34 @@ module github.com/ONSdigital/dp-mongodb-in-memory
 go 1.17
 
 require (
```

Comment:
can all dependencies be defined into one require-block

Reply:
From Go 1.17 direct and indirect deps are kept in two different blocks.  See https://tip.golang.org/doc/go1.17
> Because the number of explicit requirements may be substantially larger in an expanded Go 1.17 go.mod file, the newly-added requirements on indirect dependencies in a go 1.17 module are maintained in a separate require block from the block containing direct dependencies.